util/vector_iterator.h

```diff
@@ -20,7 +20,7 @@ class VectorIterator : public InternalIterator {
       : keys_(std::move(keys)),
         values_(std::move(values)),
         indexed_cmp_(icmp, &keys_),
-        current_(keys.size()) {
+        current_(keys_.size()) {
```

Comment:
To preserve the previous behavior, we can set it to 0. (assuming that keys is empty after std::move).

Reply:
Can confirm. `keys` is guaranteed to be empty after the move.